src/service/file_manager/file_manager.service.ts

```diff
@@ -23,11 +36,39 @@ export class FileManagerService {
     return folder;
   }
 
-  public async getContent(user: User, location: string): Promise<{ files: Array<File>; folders: Array<Folder> }> {
-    const prefix = `organization/${user.id}/user/${user.id}`;
-    const wLocation = location.endsWith('/') ? location : `${location}/`;
-    const userLocation = path.join(prefix, wLocation);
+  public async uploadFiles(user: User, body: UploadFileRequest): Promise<UploadFileResponse> {
+    const links = Promise.all(
+      body.fileInfos.map(async (fileInfo) => {
+        const file = FileFactory.from({
+          userId: user.id,
+          organizationId: user.id,
+          folder: body.location,
+          filename: fileInfo.name,
+          size: fileInfo.sizeL,
+        });
+        await this.storageService.upload(file.key);
+        return { name: fileInfo.name, url: (await this.storageService.getSignedPostUrl(fileInfo.name, file.size)).url };
+      })
+    );
+
+    return { links: await links };
+  }
+
+  public async deleteFolder(user: User, body: DeleteFolderRequest): Promise<DeleteFolderResponse> {
+    const userLocation = FileManagerService.getUserLocation(user, body.path);
+    this.storageService.deleteFolder(userLocation);
+    return { parentPath: path.dirname(userLocation) } as DeleteFolderResponse;
```

Comment:
Why do you use cast here? We can use cast only when we have type `any` and we want to set a type for variable